src/Handler/Exception/ExceptionResolver.php

```diff
@@ -25,6 +25,12 @@ public static function resolveException(\Throwable $e): \Throwable
                 if (strpos($e->getMessage(), 'No more room in database') !== false) {
                     return NoSpaceException::createForFullDB($e);
                 }
+                if (strpos($e->getMessage(), "available data bytes in the table's perm row has exceeded")) {
+                    return RowSizeTooLargeException::createForTablePermRowHasExceededLimit($e);
+                }
+                if (strpos($e->getMessage(), 'Table has too many columns')) {
```

Comment:
dejme tomu, ze to tady v tom resolveru nechame. Tak prosim at ta message je aspon shodna s message v ColumnsCollection

Reply:
upraveno